packages/components/src/FormField/FormField.css

```diff
@@ -329,6 +329,7 @@
 
 .affixIcon.suffix {
   margin: 0 0 0 calc(var(--field--padding-right) * -1);
+  padding-left: var(--space-small);
```

Comment:
Is there a reason we _wouldn't_ do the same thing for the padding-right on the default `affixIcon`?

Reply:
Played with this in product, feels like it needs more work to look good in the prefix context